index.html

```diff
@@ -365,13 +335,13 @@ <h2 class="title2 desktop">Contact me</h2>
             If you have an application you are interested in developing, a feature that you need built or a project that
             needs coding. I’d love to help with it
             </p>
-            <div class="contact-form">
+            <div class="contact-form" >
               <form class="action-form" action="https://formspree.io/f/{form_id}" method="post">
+                <label for="name" class="form-label">Name:</label>
                 <input type="text" class="label first" id="name" name="firstname" placeholder="Yeremias" maxlength="30" required>
-                <input type="text" class="label desktop" id="name" name="firstname" placeholder="Alexandra Loremipsum" maxlength="30" required>
+                <label for="email" class="form-label">Email:</label>
                 <input type="email" class="label email" id="email" name="email" placeholder="natajayanji4@gmail.com"  required>
-                <input type="email" class="label desktop" id="email" name="email" placeholder="alexandra@gmail.com"  required>
-
+                <label for="subject" class="form-label">Subject:</label>
                 <textarea class="text-area" id="subject" name="subject" placeholder="Write your message here"  maxlength="500" required ></textarea>
```

Comment:
Fixed interaction accessibility, Added form labels, and hide them to match the Figma design